cmd/lncli/cmd_payments.go

```diff
@@ -1371,12 +1371,16 @@ func forwardingHistory(ctx *cli.Context) error {
 		maxEvents = uint32(m)
 	}
 
+	// By default we will look up the peers' alias information unless the
+	// skip_peer_alias_lookup flag is specified.
+	lookupPeerAlias := !ctx.Bool("skip_peer_alias_lookup")
```

Comment:
since the whole idea is to make this opt-in now, should we not keep the lncli API the same and then have an `include-peer-aliases` flag to opt in rather?

Reply:
Thx for the review @ellemouton. I argued in the PR description that for convencience's sake we still want the `lncli` API to include the peer aliases per default. The peer alias on a forwarding event is meaningful to me so I guess that other users have same expectations and it would be nice if it could appear in the history without extra effort/flags.

Reply:
cool, makes sense :)